images/scripts/rhel.setup

```diff
@@ -291,6 +291,19 @@ fi
 pkgs="$TEST_PACKAGES $COCKPIT_DEPS $IPA_CLIENT_PACKAGES"
 $YUM_INSTALL $pkgs
 
+# If the insights-client is installed, update the insights-client
+# "egg" and make sure that only the newest one is used.
+
+if [ -x /usr/bin/insights-client ]; then
+    insights-client --status >/dev/null
+    if [ -f /var/lib/insights-client/newest.egg ]; then
+        cp /var/lib/insights-client/newest.egg /var/lib/insights-client/last_stable.egg
+        cp /var/lib/insights-client/newest.egg.asc /var/lib/insights-client/last_stable.egg.asc
+        rm -f /etc/insights-client/rpm.egg /etc/insights-client/rpm.egg.asc
+    fi
+    insights-client --version
```

Comment:
Is that the bit that will download the newest "egg"? (If so, what a confusing option -- GNU commands just show their software version with that)

Reply:
No, `insights-client --status` does the download.  Every real operation does that (other than --version and --help), and `--status` is just a convenient one since it doesn't make any changes anywhere.